apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConversionOptions.cs

```diff
@@ -251,5 +219,30 @@ private void ValidateAndParseClrToSpannerTypeMappings(string input, SpannerConve
                 }
             }
         }
+
+        private static void CheckMappingsFormat(string[] mappings, string mappingType)
```

Comment:
It took me a while to work out the purpose of this - and it looks like we always use this just before CheckValidMappings. Maybe combine the two, and document it as "validates that 'mappings' contains at least one mapping, and all mappings are valid"? (Or I may have missed something...)

Reply:
Yes, done.